### PR TITLE
Added missing `Options` from `@hey-api/client-fetch`

### PR DESCRIPTION
**Pull request by onurgenes**
Added missing `@hey-api/client-fetch` which was causing non typed query options

### CODE REVIEW

**Comment by tconroy** [comment]
+1 would be great to get this merged in. 

Also, if you're using the `axios` client, it expects a similar Options import:
```
import { Options } from '@hey-api/client-axios';
```

File: src/createImports.mts [hunk 1]
```diff
@@ -45,7 +45,23 @@ export const createImports = ({
           ts.factory.createImportSpecifier(
             true,
             undefined,
-            ts.factory.createIdentifier("QueryClient"),
+            ts.factory.createIdentifier("Options")
+          ),
+        ])
+      ),
+      ts.factory.createStringLiteral("@hey-api/client-fetch"),
```
Comment (7nohe) [inline src/createImports.mts:52]: Could you consider importing `@hey-api/client-fetch` or `@hey-api/client-axios` depending on the `client` option?

**Review by 7nohe — CHANGES_REQUESTED** [review]
Please also update the test code. Thanks!

File: src/createImports.mts (hunk 1 above)
Comment (onurgenes) [reply src/createImports.mts]: added the check with ternary. it should be OK now.

**Review by 7nohe — APPROVED** [review]
I appreciate your work. It looks good.

**Comment by onurgenes** [comment]
Awesome. Thank you for the repo @7nohe

**Pull request merged**